types/index.ts

```diff
@@ -12,5 +12,6 @@ declare global {
     VERDACCIO_SCOPE: string;
     VERDACCIO_VERSION: string;
     VERDACCIO_API_URL: string;
+    VEDACCIO_LANGUAGE: string;
```

Comment:
Reminder, this should be part of `__VERDACCIO_BASENAME_UI_OPTIONS` and `VerdaccioOptions` as `language`.